frontend/public/components/utils/kebab.tsx

```diff
@@ -484,7 +483,7 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
       isDisabled={
         isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')
       }
-      hoverMessage={hoverMessage ? hoverMessage : t('public~Resource is being deleted.')}
+      hoverMessage={_.get(resource.metadata, 'deletionTimestamp') ? hoverMessage : null}
```

Comment:
FYI The hover message is still shown on the SS page if it's progressing.

![image](https://user-images.githubusercontent.com/33557095/195391487-25d95077-00e5-4eac-bb69-b6f871b51441.png)